Remora.Discord.Interactivity/Responders/InteractivityResponder.cs

```diff
@@ -149,13 +148,7 @@ or ComponentType.MentionableSelect
         var buildParameters = data.ComponentType switch
         {
             ComponentType.Button => new Dictionary<string, IReadOnlyList<string>>(),
-            ComponentType.StringSelect => Result<Dictionary<string, IReadOnlyList<string>>>.FromSuccess
-            (
-                new Dictionary<string, IReadOnlyList<string>>
-                {
-                    { "values", data.Values.Value }
-                }
-            ),
+            ComponentType.StringSelect when data.Values.Value.IsT1 => BuildParametersFromStringData(data),
```

Comment:
StringSelect must be able to handle both union types.
As there might be a `StringSelect` component with only unsigned number options.
Those would be handled kinda invalidly as `Snowflake`. Therefore `StringSelect` must be able to build the parameters for `string` and `Snowflake`.

Reply:
Ah great point, thanks! I've updated the logic correspondingly